python/nano/src/bigdl/nano/pytorch/low_precision/jit_int8_model.py

```diff
@@ -0,0 +1,203 @@
+            return
+        
+        self.original_state_dict = model.state_dict()
+        self.channels_last = channels_last
+        self.jit_strict = jit_strict
+        self.jit_method = jit_method
+        self.enable_onednn = enable_onednn
+        self._nano_context_manager = generate_context_manager(accelerator="jit",
+                                                              precision="int8",
+                                                              thread_num=thread_num,
+                                                              enable_onednn=enable_onednn)
+        self.thread_num = thread_num
+        self.original_model = model
+        if self.channels_last:
+            self.model = self.model.to(memory_format=torch.channels_last)
+
+        if q_config is None:
+            self.q_config = get_default_qconfig_mapping("fbgemm")
+        else:
+            if isinstance(q_config, QConfig):
+                self.q_config = {'' : q_config}
+            else:
+                self.q_config = q_config
+        
+        if input_sample is None:
+            input_sample = next(iter(calib_data))
+            if isinstance(input_sample, (tuple, list)) and len(input_sample) > 1:
+                input_sample = input_sample[0]
+                if self.channels_last:
+                    if isinstance(input_sample, torch.Tensor):
+                        input_sample = input_sample.to(memory_format=torch.channels_last)
+                    else:
+                        input_sample = tuple(map(lambda x: x.to(memory_format=torch.channels_last),
+                                                 input_sample))
+
+        self.model = prepare_fx(self.model, self.q_config,
+                                example_inputs=(input_sample,))
+
+        for x in calib_data:
+            if isinstance(x, (tuple, list)) and len(x) > 1:
+                x = x[0]
+            if isinstance(x, Sequence):
+                self.model(*x)
+            else:
+                self.model(x)
+
+        self.model = convert_fx(self.model)
+
+        with torch.no_grad():
+            if self.jit_method == 'trace':
+                self.model = torch.jit.trace(self.model, input_sample,
+                                             strict=jit_strict)
+            elif self.jit_method == 'script':
+                self.model = torch.jit.script(self.model)
+            else:
+                try:
+                    self.model = torch.jit.trace(self.model, input_sample,
+                                                 strict=jit_strict)
+                except Exception:
+                    self.model = torch.jit.script(self.model)
+            self.model = torch.jit.freeze(self.model)
+            
+    @property
+    def forward_args(self):
+        return [input_value.debugName() for input_value in self.model.graph.inputs()
+                if not input_value.debugName().startswith('self')]
```

Comment:
where do we need this `forward_args`?

Reply:
This is to keep consistent with ipex inference models, but I didn't know what it is for exactly. If it's just an ipex feature, I will remove this.

Reply:
has removed

Reply:
has been removed